{{ cookiecutter.__package_name_kebab_case }}/Dockerfile

```diff
@@ -39,6 +31,27 @@ ENV VIRTUAL_ENV /opt/{{ cookiecutter.__package_name_kebab_case }}-env
 # Set the working directory.
 WORKDIR /workspaces/{{ cookiecutter.__package_name_kebab_case }}/
 
+FROM base as builder
```

Comment:
Based on my review, I think it might be more clear to call this stage `poetry`, as that's what defines this stage to me: it makes Poetry available to install (and compile, if needed) packages. Would that be OK, or do you prefer a different name?